source/use-cases/sensitive-data-encryption.txt

```diff
@@ -124,7 +124,153 @@ In this step, we generate a local master key to expedite setup of our developmen
           }
 
 
-B. Define a JSON Schema
+B. Create a Data Encryption Key
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this section, we use the MongoDB driver to create and store a data
+encryption key, used by CSFLE to automatically encrypt and decrypt data on
+document fields. The following configuration values are required by the client
+to generate a new data encryption key:
+
+* The local master key or AWS KMS master key access settings.
+* A MongoDB connection string that authenticates on a running server.
+* The key vault database and collection name.
+
+Upon creating a data encryption key, the driver returns the ``_id`` field
+(as a `UUID <https://en.wikipedia.org/wiki/Universally_unique_identifier>`_)
+of a document that stores information about the key including:
```

Comment:
Are we certain that this is a UUID? I would imagine that it's a 12-byte ObjectId value unless this specifically does not use the default.

Reply:
I might interpret this as "the _id field stores information about the key" rather than the document. I think we can also explicitly say that the driver:

1.  creates the key
2. inserts a new document into the keyvault with info about the key
3. returns the _id of the inserted document

Reply:
> Why.....
I'd imagine this has to do with the crypto library and someone's decision not to convert it to an ObjectId. It does not seem that important to the user, however, since it would only be defined in the JSON Schema.

Per the second comment, I will try to reformat as an ordered list.